src/main/java/emissary/client/EmissaryResponse.java

```diff
@@ -117,5 +119,15 @@ private <T extends BaseEntity> T makeErrorEntity(String msg, Class<T> mapper) {
         return r;
     }
 
+    public static class EmissaryResponseResponseHandler implements HttpClientResponseHandler<EmissaryResponse> {
```

Comment:
Not that it matters, but why not just `EmissaryResponseHandler`?

Reply:
Because that would make too much sense :P

Reply:
Done. Updated to ResponseEmissaryResponseResponseHandlerResponse.